recipes/pyjwkest/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyjwkest" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 128e3c81d02993ac4cd7e29ef7aac767d91daa59380e6883ae589092945e4aad
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - future
+    - pycryptodomex
+    - python
+    - requests
+    - six
+
+test:
+  imports:
+    - jwkest
+  requires:
```

Comment:
As in the other recipe, I don't think the `requires` section should be necessary.